src/app/space/create/deployments/apps/deployments-apps.component.spec.ts

```diff
@@ -14,6 +14,8 @@ import {
   TestContext
 } from 'testing/test-context';
 
+import { Filter, FilterEvent } from 'patternfly-ng/filter';
```

Comment:
Filter is unused and can be removed

Reply:
Nice find. I've removed it as well as another unused field in the component.